test/int/databaseuser_unprotected_test.go

```diff
@@ -816,6 +816,9 @@ func normalize(user mongodbatlas.DatabaseUser, projectID string) mongodbatlas.Da
 	}
 	user.GroupID = projectID
 	user.Password = ""
+	if user.OIDCAuthType == "" {
+		user.OIDCAuthType = "NONE"
+	}
```

Comment:
@helderjs & @igor-karpukhin This seemed to fix a persistent issue I was having with the integration tests for deployment. Not sure how this was not blocking other PRs already.

Reply:
This looks strange because this check was added in my[ previous PR](https://github.com/mongodb/mongodb-atlas-kubernetes/pull/1092/files#diff-27d9773cd6a3ab4bc5ae2e82100babae6440bef8ce9db36af14cdc8d9391fd8eR811) and it's merged. You are duplicating it, see line 811

Reply:
oh, I get it now. This is what happened:

1. The issue was there.
2. You fixed it in your PR.
3. I fixed it in mine.
4. I rebased again, so I got it fixed twice.

Will fix it now.